tools/make/build-container.mk

```diff
@@ -34,7 +34,7 @@
 # variable names should be passed through to the container.
 
 USE_CONTAINER       ?= 0
-BUILD_IMAGE_VERSION ?= v0.1.0
+BUILD_IMAGE_VERSION ?= v0.1.1
```

Comment:
There's also references in Dockerfile and Dockerfile.windows, can you update those as well?

(You'll also have to run `make drone` again to rebuild the Drone)

Reply:
yea